src/lando/ui/legacy/api.py

```diff
@@ -40,15 +39,16 @@ def post(self, request):
 
         if not is_user_authenticated():
             return JsonResponse(
-                {"errors":
-                    ["You must be logged in to request an uplift"]}, status_code=401)
+                {"errors": ["You must be logged in to request an uplift"]},
+                status_code=401,
+            )
 
         if not uplift_request_form.validate():
             errors = list(chain(*uplift_request_form.errors.values()))
             return JsonResponse({"errors": errors}, status_code=400)
 
-        revision_id = uplift_request_form.revision_id.data
-        repository = uplift_request_form.repository.data
+        # revision_id = uplift_request_form.revision_id.data
+        # repository = uplift_request_form.repository.data
```

Comment:
Unrelated?

Reply:
They are unused variables that popped up in the lint check. There's additional commented out code below that is connected to these variables as well.